releasenotes/notes/use-dd-namespaced-fields-in-otlp-receiver-2924efdc1fed024f.yaml

```diff
@@ -0,0 +1,14 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Adds the functionality to construct a DD span from datadog. attributes on an incoming span. By default, look for a span attribute starting with datadog.; if it's present, it's used to compute the corresponding datadog field (e.g., datadog.service is used to set ddspan.Service). It will override other sources of the same field (e.g., if both datadog.service and service.name are present, datadog.service is used).
```

Comment:
```suggestion
    Adds functionality to construct a DD span from `datadog.` attributes on an incoming span. By default, it checks for a span attribute starting with `datadog.`; if present, it uses this to compute the corresponding Datadog field (e.g., `datadog.service` is used to set `ddspan.Service`). This will override other sources for the same field (e.g., if both `datadog.service` and `service.name` are present, `datadog.service` is used).
```